pom.xml

```diff
@@ -109,52 +106,52 @@
 
 	<build>
 		<plugins>
-				<plugin>
-    <groupId>org.jacoco</groupId>
-    <artifactId>jacoco-maven-plugin</artifactId>
-    <version>0.8.2</version>
-    <executions>
-    
-    <execution>
-    <id>jacoco-check</id>
-    <goals>
-        <goal>check</goal>
-    </goals>
-    <configuration>
-        <rules>
-            <rule>
-                <element>PACKAGE</element>
-                <limits>
-                    <limit>
-                        <counter>LINE</counter>
-                        <value>COVEREDRATIO</value>
-                        <minimum>0.50</minimum>
-                    </limit>
-                </limits>
-            </rule>
-        </rules>
-    </configuration>
-</execution>
+			<plugin>
+			    <groupId>org.jacoco</groupId>
+			    <artifactId>jacoco-maven-plugin</artifactId>
+			    <version>0.8.2</version>
+			    <executions>
+
+			    <execution>
+			    <id>jacoco-check</id>
+			    <goals>
+				<goal>check</goal>
+			    </goals>
+			    <configuration>
```

Comment:
Please, add exclusions for classes that do not required unitest such as cnfig, dto, models, etc.